include/platform/clinician_input.h

```diff
@@ -13,3 +13,13 @@
  *
  */
 status_t get_respiratory_rate(uint32_t * RR);
+
+/*
+ * Updates <selected inspiratory time> variable insp_time 
+ * when a valid value was read from the clinician inputs
```

Comment:
I don't think that this description actually describes what the intention of the method is... is it not just a getter? The comment might be interpreted to mean that the value of the clinician input is actually read from the physical knobs/dials. however, I think what is meant is that it returns the last value that was input by the clinician.

Reply:
It is possible that no status was ever set in this case we sould get status errpr